app/controllers/pokemons_controller.rb

```diff
@@ -22,6 +22,7 @@ def edit
   # POST /pokemons or /pokemons.json
   def create
     @pokemon = Pokemon.new(pokemon_params)
+    @pokemon.image.attach(params[:image])
```

Comment:
It would be nice to refactor this out into its own method.

Reply:
Just finish refactor this out into it own function